docker/sentry.conf.py

```diff
@@ -51,7 +51,24 @@
             "PASSWORD": (env("SENTRY_DB_PASSWORD") or env("POSTGRES_ENV_POSTGRES_PASSWORD") or ""),
             "HOST": postgres,
             "PORT": (env("SENTRY_POSTGRES_PORT") or ""),
-        }
+        },
+        # TODO(hybrid-cloud): Add config vars for control silo configuration
+        "control_silo": {
+            "ENGINE": "sentry.db.postgres",
+            "NAME": (env("SENTRY_DB_NAME") or env("POSTGRES_ENV_POSTGRES_USER") or "control_silo"),
+            "USER": (env("SENTRY_DB_USER") or env("POSTGRES_ENV_POSTGRES_USER") or "control_silo"),
+            "PASSWORD": (env("SENTRY_DB_PASSWORD") or env("POSTGRES_ENV_POSTGRES_PASSWORD") or ""),
+            "HOST": postgres,
+            "PORT": (env("SENTRY_POSTGRES_PORT") or ""),
+        },
+        "region_silo": {
+            "ENGINE": "sentry.db.postgres",
+            "NAME": (env("SENTRY_DB_NAME") or env("POSTGRES_ENV_POSTGRES_USER") or "postgres"),
```

Comment:
Should these environment variables be different? Historically the default database name has been `sentry` as well.

Reply:
I copied this from the config above but was a bit unclear on the precedence order of all the configurations out there. I do agree that it is confusing to add more things here that don't match existing defaults. 
* Where is that sentry database set?
* Is it right that the order goes:
`~/.sentry/sentry.conf.py` > `getsentry.conf.settings.<env>` > `getsentry.conf.settings.defaults` > `sentry/docker/sentry.conf.py` > `sentry/src/sentry/conf/server.py`.